remappers/hostmetrics/process.go

```diff
@@ -289,8 +289,10 @@ func addProcessResources(resource pcommon.Resource) func(pmetric.NumberDataPoint
 		if cmdline.Str() != "" {
 			dp.Attributes().PutStr("system.process.cmdline", cmdline.Str())
 		}
-		//Adding dummy value to process.state as "undefined", since this field is not
-		//available through Hostmetrics receiver currently and Process tab in Curated UI's need this field as a prerequisite.
+		dp.Attributes().PutStr("system.process.cpu.start_time", startTimeStr)
```

Comment:
So, we are adding it as a string in the end.
this works fine for ES path, since we have the mapping of "date" hence the data shows up.
Do we have similar mapping in APM path as well ?

Reply:
We use dynamic mapping which would map a string to a date if it passes data detection ([ref](https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-field-mapping.html))